google-datacatalog-qlik-connector/setup.py

```diff
@@ -40,6 +40,7 @@
     install_requires=(
         'google-datacatalog-connectors-commons >= 0.6.0',
         'requests_ntlm',
+        'websockets',
     ),
     setup_requires=('pytest-runner',),
     tests_require=('pytest-cov',),
```

Comment:
Please check the `classifiers` on this file:

```
    classifiers=[
        release_status,
        'Natural Language :: English',
        'Programming Language :: Python',
        'Programming Language :: Python :: 3.6',
        'Programming Language :: Python :: 3.7',
        'Programming Language :: Python :: 3.7',
    ]
```
the line with `Python :: 3.7` should be `Python :: 3.8 right`?

Also, since you are using `async` and `await`... They became reserved keyword on `Python 3.7+` should you drop support from `Python 3.6` ?